MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.m

```diff
@@ -129,10 +129,16 @@ - (void)attachAVPlayer:(AVPlayer *)player {
               forKeyPath:@"status"
                  options:NSKeyValueObservingOptionInitial | NSKeyValueObservingOptionNew
                  context:MUXSDKAVPlayerStatusObservationContext];
+    
+    [[NSNotificationCenter defaultCenter] addObserver:self
```

Comment:
Nit: for better readability, we should group all the notification subscriptions together. The others start at line 147.

Reply:
Let me test something first, but I'll probably do this

Reply:
ok, it was fine. Change made 👍